README.md

```diff
@@ -102,6 +103,11 @@ To get the MAC address of the network license manager:
 
 **Note**: `Network Drive (W:)` is mapped to: `\\appstorage<uniqueID>.file.core.windows.net\webapps`.
 
+### Linux Virtual Machine
+1. Remotely connect to the server VM. For details, see [How do I remotely connect to the server virtual machine?](#how-do-i-remotely-connect-to-the-server-virtual-machine).
+1. Copy your app to the folder `/mnt/webapps/apps` using scp with the command format `scp <path/to/webapp> <username>@<virtualMachineIP>:/mnt/webapps/apps`.
```

Comment:
It may help to add that /mnt/webapps/apps is on the server VM and path/to/webapp is on the machine where the web app to copy is located. Does SCP need to be in caps when you are referring to it as a protocol?